secator/utils.py

```
@@ -1426,3 +1426,46 @@ def vhs_tap_to_tape(tap_file, output_tape, width=None, height=None, font_size=12
+			return item._compare_key()
+		# Plain dict: try to load into the appropriate OutputType to use _compare_key()
+		if isinstance(item, dict):
+			from secator.output_types import OUTPUT_TYPES
+			_type = item.get('_type')
+			if _type:
+				for cls in OUTPUT_TYPES:
+					if cls.get_name() == _type:
+						try:
+							return cls.load(item)._compare_key()
+						except Exception:
+							pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)